app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 50 }
+
+  validates :title, uniqueness: true
+
+  validates :body, presence: true, length: { maximum: 50 }
+
+  validates :body, uniqueness: true
+
+  belongs_to :user
+
+  has_many :comment
```

Comment:
This relationship has some issues. Check below :arrow_down: 

- [ ] A post has many comments not comment